packages/server/src/factories/variant.ts

```diff
@@ -12,8 +12,11 @@ export default Factory.define<EpicVariant>(() => ({
     ],
     highlightedText:
         'Support The Guardian from as little as %%CURRENCY_SYMBOL%%1 - and it only takes a minute. Thank you.',
-    backgroundImageUrl:
-        'https://images.unsplash.com/photo-1494256997604-768d1f608cac?ixlib=rb-1.2.1&auto=format&fit=crop&w=1701&q=80',
+    image: {
+        mainUrl:
+            'https://images.unsplash.com/photo-1494256997604-768d1f608cac?ixlib=rb-1.2.1&auto=format&fit=crop&w=1701&q=80',
+        altText: 'An image of a cat',
```

Comment:
I'm worried about hardcoding a cat image URL here - is it possible that this image could ever appear on the main site? If yes, there may be legal implications around sourcing the image from a 3rd party - unless we've bought a licence to use the image in which case, why isn't it in the Grid?

Reply:
it's only used for testing + storybook, but I've replaced with an image from the Grid!